.github/workflows/atlantis-image.yml

```diff
@@ -57,10 +57,18 @@ jobs:
         labels: |
           org.opencontainers.image.licenses=Apache-2.0
         tags: |
-          type=ref,event=branch
-          type=ref,event=pr
-          type=semver,pattern={{version}}
-          type=semver,pattern={{major}}.{{minor}}
+          type=ref,event=pr,suffix=${{ format('-{0}', matrix.image_type) }}
+          type=semver,pattern={{version}},suffix=${{ format('-{0}', matrix.image_type) }}
+          type=semver,pattern={{major}}.{{minor}},suffix=${{ format('-{0}', matrix.image_type) }}
+          type=sha,suffix=${{ format('-{0}', matrix.image_type) }}
+          type=raw,event=push,value=dev,enable=${{ github.ref == format('refs/heads/{0}', 'main') && matrix.image_type == 'alpine' }}
+          type=raw,event=push,value=dev,enable={{is_default_branch}},suffix=${{ format('-{0}', matrix.image_type) }}
+          type=raw,event=tag,value=prerelease-latest,enable=${{ startsWith(github.ref, 'refs/tags/') && contains(github.ref, 'pre') && matrix.image_type == 'alpine' }}
+          type=raw,event=tag,value=prerelease-latest,enable=${{ startsWith(github.ref, 'refs/tags/') && contains(github.ref, 'pre') }},suffix=${{ format('-{0}', matrix.image_type) }}
+          type=raw,event=tag,value=latest,enable=${{ startsWith(github.ref, 'refs/tags/') && !contains(github.ref, 'pre') && matrix.image_type == 'alpine' }}
+          type=raw,event=tag,value=latest,enable=${{ startsWith(github.ref, 'refs/tags/') && !contains(github.ref, 'pre') }},suffix=${{ format('-{0}', matrix.image_type) }}
+        flavor: |
+          latest=false
```

Comment:
Could use suffix within flavor and remove the suffix from other tags. We'd just need to ensure that we can empty the suffix for the `dev` tag on alpine